andela/codango
# #119258079 Navbar Synced navigation

#### What does this PR do?
Ensures that `Pair Program` tab is highlighted when the user is on the Pair program page
#### Description of Task to be completed?
Set the tab to active according to the page you are on
#### How should this be manually tested?
Once logged in to Codango, click on `Pair Program` on the navbar. The highlighed tab should be *Pair Program*
#### Any background context you want to provide?
Before this fix `Community` tab was active on the *Pair Program* page
#### What are the relevant pivotal tracker stories?
#119258079
#### Screenshots
<img width="782" alt="screenshot 2016-05-11 18 23 26" src="https://cloud.githubusercontent.com/assets/15629602/15186414/9e178fa2-17a5-11e6-95db-48b80e0a5735.png">
